kyaml/yaml/fns.go

```diff
@@ -796,6 +803,12 @@ func SplitIndexNameValue(p string) (string, string, error) {
 	return parts[0], parts[1], nil
 }
 
+// IsMatchEveryIndex returns true if p is matching every elements.
+// e.g. "*"
+func IsMatchEveryIndex(p string) bool {
```

Comment:
Suggest changing the name here to `IsWildcard` and moving this up to line 791 to keep the `IsXXXX` functions together